VisualPinball.Engine/Math/Color.cs

```diff
@@ -17,6 +17,14 @@ public class Color
 
 		public int Bgr => Blue * 65536 + Green * 256 + Red;
 
+		public Color(Color other)
+		{
+			Red = other.Red;
+			Green = other.Green;
+			Blue = other.Blue;
+			Alpha = other.Alpha;
+		}
+

```

Comment:
I usually use a `.Clone()` method that returns a copy, but it's fine if you prefer explicitly instantiating it.

Reply:
I don't have a preference either way, i'll convert to clone methods